api/src/certification/enrolment/domain/models/timeline/CandidateCreatedEvent.js

```diff
@@ -0,0 +1,12 @@
+// @ts-check
+import { TimelineEvent } from './TimelineEvent.js';
+
+export class CandidateCreatedEvent extends TimelineEvent {
+  /**
+   * @param {Object} props
+   * @param {Date} props.when
```

Comment:
```suggestion
   * @param {Date} props.date
```

**detail**: tous les `when` auraient pu être des `date`